TASVideos/Pages/Shared/_LoginPartial.cshtml

```diff
@@ -29,7 +29,12 @@
 				<button id="Logout" type="submit" class="btn btn-link navbar-btn nav-link text-nowrap fs-6 border-0"><i class="fa fa-sign-out"></i> Log out</button>
 			</nav-item>
 			<nav-item activate="Search">
-				<a id="Search" class="nav-link text-nowrap" asp-page="/Search/Index"><i class="fa fa-search"></i> Search</a>
+				<form method="GET" action="/Search/Index">
+					<div class="input-group mt-1">
+						<button type="submit"><i class="fa fa-search"></i></button>
```

Comment:
Maybe this and the other button could use a better style, like:
`<button type="submit" class="btn btn-sm border border-secondary nav-link">`
It would fit better with the other design. It would turn this:
![image](https://user-images.githubusercontent.com/22375320/159992588-5a418a7e-cfec-4870-b767-13a09ddb8b87.png)
into this
![image](https://user-images.githubusercontent.com/22375320/159992557-3a79f652-1302-4a80-9643-290d872bb1e6.png)